src/Form/ConfigForm.php

```diff
@@ -14,16 +14,53 @@
 
 namespace Drupal\collabora_online\Form;
 
+use Drupal\collabora_online\Cool\CollaboraDiscoveryFetcherInterface;
+use Drupal\collabora_online\Exception\CollaboraNotAvailableException;
+use Drupal\Core\Config\ConfigFactoryInterface;
 use Drupal\Core\Form\ConfigFormBase;
 use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Messenger\MessengerInterface;
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+use Drupal\Core\StringTranslation\TranslationInterface;
+use Drush\Commands\AutowireTrait;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
 
 /**
  * Form to configure module settings for Collabora.
  */
 class ConfigForm extends ConfigFormBase {
 
+  use AutowireTrait;
+
   const SETTINGS = 'collabora_online.settings';
 
+  /**
+   * {@inheritDoc}
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The factory for configuration objects.
+   * @param \Drupal\Core\Config\TypedConfigManagerInterface $typedConfigManager
+   *   The typed config manager.
+   * @param \Drupal\Core\StringTranslation\TranslationInterface $string_translation
+   *   The translation service.
+   * @param \Drupal\Core\Messenger\MessengerInterface $messenger
+   *   The messenger service.
+   * @param \Drupal\collabora_online\Cool\CollaboraDiscoveryFetcherInterface $discoveryFetcher
+   *   Service to load the discovery.xml from the Collabora server.
+   */
```

Comment:
Can we drop the phpdoc for the constructor?
Also, it would not really make sense to have `@inheritdoc` here.